lib/enum/basic_enum.rb

```diff
@@ -22,48 +22,74 @@ def number_of_elements_in_collection
   # This one also uses 'count'
   # Try it yourself!
   def number_of_floats_or_fixnums
+    # collection
+
+    # Version 1, If using variables:
+    # new_enum = collection.map { |item| item.class}
+    # total = new_enum.count(Fixnum) + new_enum.count(Float)
+
+    # Version 2, .map vs .select
+    # new_arr2 = collection.map { |item| item.is_a? (Numeric)} #=> [true, true, false, true, true, false, false, true, true, false, true]
+    # new_arr3 = collection.select { |item| item.is_a? (Numeric)} #=> [1, 2, 3, 4, 5, 6, 3.14]
+
+    collection.select { |item| item.is_a? (Numeric)}.count
   end
 
   # Refer to: all?
   # http://ruby-doc.org/core-2.1.4/Enumerable.html#method-i-all-3F
   def all_words_longer_than_length?(min_length)
+    # ea_word_len = collection.all? { |item| item.length > min_length}
```

Comment:
Seems like you don't need that commented-out code anymore, so you can delete it. (Notes to yourself are fine, but this doesn't seem to be one of those.)